pallets/identity/src/lib.rs

```diff
@@ -55,25 +55,27 @@ pub mod pallet {
         StorageValue<Value = u32, QueryKind = ValueQuery, OnEmpty = DefaultCurrent<T>>;
 
     #[pallet::storage]
+    #[pallet::unbounded]
     #[pallet::getter(fn identity_list)]
     /// Maps accounts to the array of identities it owns.
     pub type IdentityList<T: Config> =
         StorageMap<_, Blake2_128Concat, T::AccountId, BTreeSet<u32>, ValueQuery>;
 
     #[pallet::storage]
+    #[pallet::unbounded]
```

Comment:
@PhillyWebGuy @insipx I know this works, and it's probably nothing to worry about, but do we want to think through this and make sure we're okay using unbounded storage?

Reply:
Yeah this would make for a good investigation tomorrow during our call, i wonder what the implications of this might be